src/modules/helpers/index.ts

```diff
@@ -622,6 +622,41 @@ export class HelpersModule {
     return arrayCopy.slice(min);
   }
 
+  /**
+   * Returns a random value from an Enum object.
+   *
+   * This does the same as `objectValue` except that it ignores (the values assigned to) the numeric keys added for TypeScript enums.
+   *
+   * @param enumObject Enum to pick the value from.
+   * @param <EnumType> Type of generic enums, automatically inferred by TypeScript.
+   * @param <EnumValueType> Type of enumObject property, also automatically inferred by TypeScript.
```

Comment:
We use the `@template` tags for the generic parameters as this has better readability in VSCode.

```suggestion
   * @template EnumType Type of generic enums, automatically inferred by TypeScript.
   * @template EnumValueType Type of enumObject property, also automatically inferred by TypeScript.
   * @param enumObject Enum to pick the value from.
```